src/cascadia/TerminalApp/TerminalWindow.cpp

```diff
@@ -1322,6 +1324,44 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    void TerminalWindow::_WindowSizeChanged(const IInspectable&, winrt::Microsoft::Terminal::Control::WindowSizeChangedEventArgs args)
+    {
+        til::size cellCount{ args.Width(), args.Height() };
+        const auto dpi = GetDpiForWindow(_hwnd);

```

Comment:
This is a temporary workaround that I had suggested. I think it's better to use `GetDpiForWindow` here because it more closely mirrors other, related code that also uses `GetDpiForWindow`. I'd like to clean this up all at once in the 1.23 period.